src/Fixer/PhpUnit/PhpUnitTestAnnotationFixer.php

```diff
@@ -291,26 +291,36 @@ private function getDockBlockIndex(Tokens $tokens, $index)
         return $index;
     }
 
+    /**
+     * @param string $functionName
+     *
+     * @return bool
+     */
+    private function hasTestPrefix($functionName)
+    {
+        if (!$this->startsWith('test', $functionName) || strlen($functionName) < 5) {
+            return false;
+        }
+
+        $nextCharacter = $functionName[4];
+
+        return is_numeric($nextCharacter) || $nextCharacter === strtoupper($nextCharacter);
```

Comment:
The second part covers also cases where we have `test_` as prefix, as `'_' === strotoupper('_')`.